public/checky.html

```diff
@@ -49,653 +49,45 @@
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"><input
-	type="checkbox"><input type="checkbox"><input type="checkbox"><input type="checkbox"></body>
+
+
+
+<body>
+	<div class="user-controlled">
+		<!--Placed as a div for now, considering 
+		we can add the ability to change grid size-->
+		<button onClick="resetAll()">Reset all check boxes</button>
+	</div>
+	<div id="boxes">
+
+	</div>
+
+	<script language="JavaScript">
+		totalBoxes = 2590
+		for (var i=0; i< totalBoxes; i++){
+			var element = document.createElement('input');
+			element.id = 'box';
+			element.type = 'checkbox';
+			element.checked = true;
+			document.getElementById('boxes').appendChild(element);
+		}
+	
+		function resetAll(){
+			let response = confirm('Reset all checkboxes to an UNCHECKED state?')
+			if (response){
+				for (i=0; totalBoxes - 1; i++){
+					let box = document.getElementById('boxes').getElementsByTagName('input')[i];
```

Comment:
I've found that `.item()` is better for this situation

Reply:
Give an example?

Reply:
https://github.com/zanderp25/www/blob/master/pages/checky.html.tsx#L14